### ISSUE TITLE
Python3 not found in Travis CI Apple environment

### ISSUE THREAD

**Issue by burlen** [LBL-EESA/TECA#398]
```
-- Could NOT find PythonInterp: Found unsuitable version "2.7.17", but required is at least "3" (found /usr/local/bin/python)
-- Could NOT find PythonLibs: Found unsuitable version "2.7.10", but required is at least "3" (found /usr/lib/libpython2.7.dylib)
-- Could NOT find NumPy (missing: NumPy_INCLUDE_FOUND NumPy_VERSION_FOUND) 
-- Could NOT find MPI4Py (missing: MPI4Py_INCLUDE_FOUND MPI4Py_VERSION_FOUND) 
-- Found SWIG: TRUE
CMake Error at CMakeLists.txt:269 (message):
  Python 3 features -- required but not found
```

**Comment by burlen** [comment]
@elbashandy @taobrienlbl FYI Apple dashboards are not working, and it is not because of any changes in our code base. I'll take a look. In the meantime expect false positive test failures.

**Comment by burlen** [comment]
closed by #381
